src/TestExplorer/TestExplorer.ts

```diff
@@ -19,7 +19,7 @@ import { FolderEvent, WorkspaceContext } from "../WorkspaceContext";
 import { TestRunner } from "./TestRunner";
 
 /** Build test explorer UI */
-export class TestExplorer {
+export class TestExplorer implements vscode.Disposable {
```

Comment:
Again is this necessary? Just wondering why you've added them. I guess they give more information about the class